slog.go

```diff
@@ -29,11 +29,11 @@ type Sink interface {
 	Sync()
 }
 
-// LogEntry logs the given entry with the context to the
+// Log logs the given entry with the context to the
 // underlying sinks.
 //
 // It extends the entry with the set fields and names.
-func (l Logger) LogEntry(ctx context.Context, e SinkEntry) {
+func (l Logger) Log(ctx context.Context, e SinkEntry) {
```

Comment:
It might be better to change the Sink interface instead since I assume it's implemented less often than the `log.LogEntry()` method is called.

Reply:
That's certainly not the case. We have 4-5 implementations of `slog.Sink`, one in its own repo, and zero uses of `(Logger).LogEntry` outside of tests.